spyder/utils/tests/test_dochelpers.py

```diff
@@ -0,0 +1,80 @@
+        assert getargtxt(Test.method) == ['x, ', 'y=2']
+        assert getdoc(sorted) == {'note': 'Function of __builtin__ module',
+                                  'argspec': u'(iterable, cmp=None, key=None, '
+                                              'reverse=False)',
+                                  'docstring': u'sorted(iterable, cmp=None, '
+                                                'key=None, reverse=False) --> '
+                                                'new sorted list',
+                                  'name': 'sorted'}
+        assert getargtxt(sorted) == ['iterable, ', ' cmp=None, ',
+                                     ' key=None, ', ' reverse=False']
+    else:
+        assert getargtxt(Test.method) == None
+        if os.name == 'nt':
+            assert getdoc(sorted) == {'note': 'Function of builtins module',
+                                      'argspec': '(...)',
+                                      'docstring': 'Return a new list '
+                                                   'containing '
+                                                   'all items from the '
+                                                   'iterable in ascending '
+                                                   'order.\n\nA custom '
+                                                   'key function can be '
+                                                   'supplied to customise the '
+                                                   'sort order, and '
+                                                   'the\nreverse flag can be '
+                                                   'set to request the result '
+                                                   'in descending order.',
+                                                   'name': 'sorted'}
+        else:
+            assert getdoc(sorted) == {'note': 'Function of builtins module',
+                                      'argspec': '(...)',
+                                      'docstring': 'Return a new list '
+                                                   'containing '
+                                                   'all items from the '
+                                                   'iterable in ascending '
+                                                   'order.\n\nA custom '
+                                                   'key function can be '
+                                                   'supplied to customize the '
+                                                   'sort order, and '
+                                                   'the\nreverse flag can be '
+                                                   'set to request the result '
+                                                   'in descending order.',
+                                                   'name': 'sorted'}          
+        assert getargtxt(sorted) == None
+    assert isdefined('numpy.take', force_import=True) == True
```

Comment:
There's no need to `assert ... == True`. You can change this simply to `assert ...`